src/components/dataset/query/sidebar/panels/InfoView.jsx

```diff
@@ -112,10 +111,7 @@ export class InfoView extends React.PureComponent {
                   Summary of roles and memberships
                 </AccordionSummary>
                 <AccordionDetails>
-                  <DatasetAccess
```

Comment:
On line 104 above (`<Accordion>`) would it be possible to add control to the expanded state through the dataset store? I ask because without it, I noticed an annoying behavior that when I click on help then close, it's collapsed the accordion.  It'd be great to persist it as expanded

Reply:
The accordion now stays expanded when a user clicks on the help link!

Reply:
I've deployed it on my personal instance and you can check it out: https://jade-tl.datarepo-dev.broadinstitute.org.

Side not, I just used my feature to add Jade Dev Stewards to my dataset. That's so cool!

Reply:
Hmm...I'm not seeing that works on the deployed UI but the code looks like it should work so might be a deployment thing